### ISSUE TITLE
Camel Case urls not working on some cases

### ISSUE THREAD

**Issue by washimimizuku** [balderdashy/sails#2994]
On my local machine, a Mac, when I access something like this it works fine: 

http://localhost:1337/backoffice/elementType/list

But on the my production server, which is Ubuntu 14.04, it doesn't:
```
error: Sending 500 ("Server Error") response: 
 Could not render view "backoffice/elementtype/list".  Tried locating view file @ "/home/wide/node/wide-cms/views/backoffice/elementtype/list". Layout configured as "backoffice/layout", so tried using layout @ "/home/wide/node/wide-cms/views/backoffice/layout")
```

It looks for folder backoffice/elementtype, when it should look for backoffice/elementType

**Comment by sailsbot** [comment]
Thanks for posting, @washimimizuku.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
